lib/game.rb

```diff
@@ -0,0 +1,68 @@
+class Game
+  WIN_COMBINATIONS = [
+    [0,1,2], # Top row
+    [3,4,5], # Middle row
+    [6,7,8], # Bottom row
+    [0,3,6], # Left column
+    [1,4,7], # Middle column
+    [2,5,8], # Right column
+    [0,4,8], # \
+    [6,4,2]  # /
+  ]
+
+  attr_accessor :board, :player_1, :player_2
+
+  def initialize(p1 = Player::Human.new("X"), p2 = Player::Human.new("O"), board = Board.new)
+    @player_1, @player_2, @board = p1, p2, board
+  end
+
+  def current_player
+    board.turn_count.even? ? player_1 : player_2
+  end
+
+  def over?
+    won? || draw? || full?
```

Comment:
Can the board ever be `full?` without someone having `won?` or the game being a `draw?`?